kubernetes/jobmonitor/src/main/kotlin/LostJobsFinder.kt

```diff
@@ -136,6 +160,16 @@ internal class LostJobsFinder(
         val lostJobs = jobRepository.listActive(currentTime - monitorConfig.lostJobsMinAge)
             .filterNot { it.ortRunId in kubeJobs }
 
+        return Pair(kubeJobs, lostJobs)
+    }
+
+    /**
+     * Perform actions to handle the given list of [lostJobs] for the given [endpoint].
+     */
+    private fun handleLostWorkerJobs(
+        endpoint: Endpoint<*>,
+        lostJobs: List<WorkerJob>
+    ) {
         if (lostJobs.isNotEmpty()) {
```

Comment:
This function only does some logging !? If yes, why not name it `logLostWorkerJobs` ?

Reply:
It is not. It sends notifications to Orchestrator via the `notifier`.

Reply:
Ok, I was confused because on this commit, there is only logging inside.